ElementaryTasks/Arrays.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ElementaryTasks
+{
+    class Arrays
+    {
+        public void MinElement()
```

Comment:
Please use verbs or verb phrases for method name https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members